metadata.json

```diff
@@ -22,43 +29,23 @@
       "version_requirement": ">= 2.0.0 < 4.0.0"
     }
   ],
-  "description": "Install/configure MySQL with Galera/XtraDB cluster support",
-  "license": "BSD-2-Clause",
-  "name": "fraenki-galera",
   "operatingsystem_support": [
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "8",
-        "9"
+        "8"
       ]
     },
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "16.04",
-        "18.04"
+        "16.04"
```

Comment:
Why was support for Ubuntu 18.04 removed?

Reply:
Same as above

Reply:
back in.